shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -1511,11 +1511,90 @@ - (void)setEditableTransform:(NSArray*)matrix {
   _cachedFirstRect = kInvalidFirstRect;
 }
 
+- (BOOL)transformPoint:(CGPoint)point
+              isAffine:(BOOL)isAffine
+                  minX:(CGFloat&)minX
+                  maxX:(CGFloat&)maxX
+                  minY:(CGFloat&)minY
+                  maxY:(CGFloat&)maxY {
```

Comment:
Out-parameters aren't very idiomatic in Obj-C. 

Could we return a `CGRect` instead with a sentinel value like `CGRectNull` for the case where things go wrong?

Reply:
Yeah I can use `CGRect` but since this is for computing a bounding box I think I would have to put maxX, maxY in `width` and `height`'s place.